infrastructure/lambda/template.yaml

```diff
@@ -625,6 +656,10 @@ Resources:
       TimeToLiveSpecification:
         AttributeName: expiryDate
         Enabled: true
+      SSESpecification:
+        KMSMasterKeyId: !Ref AddressTableKey
+        SSEEnabled: true
+        SSEType: KMS
```

Comment:
I think we don't want to actually apply the key at the same time as we apply the key perms to the Lambdas because that could cause a race condition where the table has the key but the Lambdas can't use it yet.